case-lib/lib.sh

```diff
@@ -293,6 +293,24 @@ find_ldc_file()
     printf '%s' "$ldcFile"
 }
 
+func_cavstool_etrace_collect()
+{
+    local logopt="-l"
+    local clogfile=$LOG_ROOT/cavstool.txt
```

Comment:
```suggestion
    local clogfile=$LOG_ROOT/etrace.txt
```
Please keep the same name for consistency. I'm working on another PR that scans for ERRORs.